applications/GeoMechanicsApplication/custom_problemtype/GeoMechanicsApplication.gid/GeoMechanicsApplication.prb

```diff
@@ -0,0 +1,231 @@
+PROBLEM DATA
```

Comment:
Problemtype folder should go into https://github.com/KratosMultiphysics/GiDInterface repo:
kratos.gid/apps/GeoMechanocs/

Reply:
Thanks, we will work on this in https://github.com/KratosMultiphysics/GiDInterface/pull/901
I think the content of the folder still needs to be restructured so it remains a draft for now